examples/tfengine/generated/team/example-prod-data/main.tf

```diff
@@ -158,7 +158,7 @@ module "example_prod_bucket" {
   source  = "terraform-google-modules/cloud-storage/google//modules/simple_bucket"
   version = "~> 1.4"
 
-  name       = "example-prod-bucket"
+  name       = "example.prod.bucket"
```

Comment:
Naming buckets with dots may be an issue. From [here](https://cloud.google.com/storage/docs/naming-buckets) and [here](https://cloud.google.com/storage/docs/domain-name-verification):
```
Names containing dots require verification. 

Cloud Storage considers bucket names that contain dots to be domain names. As such, a bucket containing a dot in its name must:

* Be a syntactically valid DNS name (for example, bucket..example.com is not valid because it contains two dots in a row).
* End with a currently-recognized top-level domain, such as .com.
* Be created by a user who is authorized to work with the domain name.

Examples of valid domain-named buckets include example.com, buckets.example.com, and www.example.co.uk
```

Reply:
Thanks for raising this! I changed the example bucket name and added a comment.

Reply:
I feel like our example bucket names should just not use dots, since there's no need for us to introduce this requirement. It seems like this is only for if you want to use your bucket as a website.

Reply:
Sure. We can keep the example just having -